modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -160,11 +158,64 @@ func (ash Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyh
 	return next.ServeHTTP(w, r)
 }
 
+func (ash Handler) getDatabaseKey() string {
+	return ash.CA
```

Comment:
Note that the CA name may not be safe for use as a filename. In CertMagic we handle this by having a sanitizing function, e.g.:

https://github.com/caddyserver/certmagic/blob/81657a23377cad89ed4c08d3741b40b3cda055c5/storage.go#L190-L209

Ours doesn't have to be that elaborate, though. Could be a simple regex probably.